app/communities-config.js

```diff
@@ -117,8 +117,8 @@ const communitiesConfig = {
   dev: {
     1: {
       origin: 'http://localhost:3000',
-      src: TelosLogo,
-      styles,
+      // src: TelosLogo,
```

Comment:
What is the reason for commenting this?

Reply:
I commented this to have current site version not community version